### DISCUSSION TITLE
maxContentWidth('full') is occuping entire page.

### DISCUSSION THREAD

**Discussion by rajenindianic** [filamentphp/filament#8613] [Help]
I am using filament 3 with laravel 10 on linux platform
when I am using maxContentWidth('full'), screen output is 

![Screenshot from 2023-09-20 11-28-41](https://github.com/filamentphp/filament/assets/94968368/c7c781da-a24d-445e-849b-94cb9914e0c3)
Will be highly obliged if someone can help me.

**Comment by leandrocfe** [comment]
Did you do this? `$panel->maxContentWidth('full')`
What is the Filament version? Did you customize something?

**Reply by otybtechnologies** [reply]
Also facing the same issue. Using filament version **^v3.0.60**

And I also did this **$panel->maxContentWidth('full')**
